lib/localisation/src/lib.rs

```diff
@@ -1 +1 @@
-#![no_std]
```

Comment:
We aren't allowed to remove the no_std or things will break

Reply:
Ah okay, can we import it to use print?